data/ships.txt

```diff
@@ -226,7 +224,8 @@ ship "Bactrian"
 	explode "large explosion" 45
 	explode "huge explosion" 30
 	"final explode" "final explosion large"
-	description "The Lionheart Bactrian is the last of the great city-ships, a design hearkening back to the early days of space colonization when a long-distance vessel needed to be a self-contained world, able to survive at times for weeks without encountering an inhabited planet. It is not only a freighter, but a carrier, and a capable warship either at short range or at a distance. Naturally, this versatility also makes it extremely expensive, and the Bactrian is not normally for sale to ordinary citizens who have not been vetted by the local government of the Deep."
+	description "The Lionheart Bactrian is the last of the great city-ships, a design hearkening back to the early days of space colonization when a long-distance vessel needed to be a self-contained world, able to survive at times for weeks without encountering an inhabited planet."
+	description "During the caravan era of the 27th century, the Bactrian was among the most powerful warships in human space, as well as being one of the best freighters and carriers. However, more modern times have seen the Bactrian become among the weakest of the warships, and it was not too many years ago that the Bactrian was recategorized as a transport vessel, and as such the Deep no longer requires for those who wish to purchase this massive ship to be vetted."
```

Comment:
I like @Pointedstick's version, but I would change it as follows:
>No longer a match for today's modern warships, the Bactrian has been reclassified as a Transport and is now sold freely to civilians.

This change makes the Bactrian's fall from being among the most powerful to being among the weakest of the human warships sound more like a gradual transition as opposed to sudden change.

Something I would also change is the "sales were restricted" part, although I can't think of anything right now. It just seems very abrupt and isn't expanded upon. Restricted to whom? (trustworthy and verified citizens) Restricted by what? (A laborious / exhaustive vetting process). It seems awkward to end the sentence on such a short and uninformative phrase.

Otherwise, it is a great change.